esm_runscripts/filelists.py

```diff
@@ -154,7 +154,7 @@ def reuse_sources(config):
                     config[model][filetype + "_sources"][categ] = (
                         config[model]["experiment_" + filetype + "_dir"]
                         + "/"
-                        + config[model][filetype + "_targets"][categ].split("/")[-1]
+                        + config[model][filetype + "_targets"][categ]
```

Comment:
I don't understand this change.

Reply:
This line allows only the reuse of resources that are files, but not folders.

Let's say for example that you want to include `input` in the reuse sources and your input has subfolders:
```
<exp_path>/input/model/subfolder/many_files
```
If you don't remove the `.split("/")[-1]` you'll end up with `many_files` directly on work, where you might want to have those files in `work/subfolder` instead. Also, this was resulting in an even more complicated and wrong behaviour when combined with the following subfolder hotfixes:
- #69 
- #72 
There, the subfolders would be copied but the files contained in each should folder would be those of the first subfolder only, or something crazy like that.

Reply:
OK, cool. thanks for explaining.